src/pe_source/data/sixgill/source.py

```diff
@@ -0,0 +1,140 @@
+"""Scripts for importing Sixgill data into PE Postgres database."""
+
+# Standard Python Libraries
+import logging
+
+# Third-Party Libraries
+import pandas as pd
+import requests
+
+from .api import (
+    alerts_count,
+    alerts_list,
+    credential_auth,
+    dve_top_cves,
+    intel_post,
+    org_assets,
+)
+
+
+def alias_organization(org_id):
+    """List an organization's aliases."""
+    assets = org_assets(org_id)
+    df_assets = pd.DataFrame(assets)
+    aliases = df_assets["organization_aliases"].loc["explicit":].tolist()[0]
+    return aliases
+
+
+def root_domains(org_id):
+    """Get root domains."""
+    assets = org_assets(org_id)
+    df_assets = pd.DataFrame(assets)
+    root_domains = df_assets["domain_names"].loc["explicit":].tolist()[0]
+    return root_domains
+
+
+def mentions(date, aliases):
+    """Pull dark web mentions data for an organization."""
+    mentions = ""
+    for mention in aliases:
+        mentions += '"' + mention + '"' + ","
+    mentions = mentions[:-1]
+    query = "site:forum_* AND date:" + date + " AND " + "(" + str(mentions) + ")"
+    logging.info("Query:")
+    logging.info(query)
+    count = 0
+    while count <= 5:
+        try:
+            logging.info("Intel post try #%s", count + 1)
+            resp = intel_post(query, frm=0, scroll=False, result_size=1)
+            count = 6
```

Comment:
Use `break` instead here to get out of the `while` loop:
```suggestion
            break
```